draft-dns-zone-digest.xml

```diff
@@ -593,7 +600,13 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
           a zone file prior to being loaded or served by a name server.
           We could allow a name server implementation to respond to
           &RRNAME; queries with the REFUSED RCODE without loss of
-          functionality.
+          functionality.  Note that refusal would prevent ensuring that a
+          zone-walk is complete. 
+        </t>
+        <t>
+	  General record modification, addition or deletion will
+	  be caught by checking either the value of the &RRNAME;
+	  or regular DNSSEC validation.
```

Comment:
This new last paragraph seems out of place in the "Attacks Utilizing the Zone Digest" section..?